.github/workflows/npm.yml

```diff
@@ -8,10 +8,10 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       # Setup .npmrc file to publish to npm
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
-          node-version: '20.x'
-          registry-url: 'https://registry.npmjs.org'
+          node-version: "20.x"
```

Comment:
Node version in `.nvm` and in `.github/workflows/main.yaml` is 18. Should this be unified?

Reply:
:heavy_check_mark: